src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        String[] color = new String[]{"red", "orange", "yellow", "green",
+                "blue", "indigo", "violet"};
+        int n = (int)Math.floor(Math.random() * color.length);

```

Comment:
```suggestion
        int index = (int) Math.random() * color.length;
```
(int) casting truncates(cuts) the remainder, so you don't have to do Math.floor